src/main/java/in/projecteka/consentmanager/SecurityConfiguration.java

```diff
@@ -104,13 +104,15 @@ public SecurityContextRepository contextRepository(SignUpService signupService,
             if (isEmpty(token)) {
                 return Mono.empty();
             }
-
             if (isSignUpRequest(exchange.getRequest().getPath().toString(), exchange.getRequest().getMethod())) {
                 return checkSignUp(token);
             }
             if (isGrantConsentRequest(exchange.getRequest().getPath().toString(), exchange.getRequest().getMethod())) {
                 return validateGrantConsentRequest(token);
             }
+            if (isRevokeConsentRequest(exchange.getRequest().getPath().toString(), exchange.getRequest().getMethod())) {
+                return validateRevokeConsentRequest(token);
+            }
```

Comment:
Can you group this together as both needs to verify the token with pinverificationservice just like what we do for centralRegistryRelated URLs?